deploy-manage/manage-connectors.md

```diff
@@ -2,25 +2,90 @@
 mapped_urls:
   - https://www.elastic.co/guide/en/kibana/current/action-types.html
   - https://www.elastic.co/guide/en/serverless/current/action-connectors.html
+applies_to:
+  stack:
+  serverless:
 ---
 
-# Manage connectors
+# Manage connectors [connector-management]
 
-% What needs to be done: Align serverless/stateful
+Connectors serve as a central place to store connection information for both Elastic and third-party systems. They enable the linking of actions to rules, which execute as background tasks on the {{kib}} server when rule conditions are met. This allows rules to route actions to various destinations such as log files, ticketing systems, and messaging tools. Different {{kib}} apps may have their own rule types, but they typically share connectors. The **{{stack-manage-app}} > {{connectors-ui}}** provides a central location to view and manage all connectors in the current space.
 
-% GitHub issue: https://github.com/elastic/docs-projects/issues/352
+::::{note}
+This page is about {{kib}} connectors that integrate with services like generative AI model providers. If you’re looking for Search connectors that synchronize third-party data into {{es}}, refer to [Connector clients](https://www.elastic.co/guide/en/serverless/current/elasticsearch-ingest-data-through-integrations-connector-client.html).
 
-% Scope notes: Connectors management, probably moving some content to reference. We should align the serverless and stateful documentation.
+::::
 
-% Use migrated content from existing pages that map to this page:
+## Required permissions [_required_permissions_2]
 
-% - [ ] ./raw-migrated-files/kibana/kibana/action-types.md
-%      Notes: 28 children
-% - [ ] ./raw-migrated-files/docs-content/serverless/action-connectors.md
+Access to connectors is granted based on your privileges to alerting-enabled features. For more information, go to [Security](../explore-analyze/alerts-cases/alerts/alerting-setup.md#alerting-security).
 
-$$$connector-management$$$
+## Connector networking configuration [_connector_networking_configuration]
 
-**This page is a work in progress.** The documentation team is working to combine content pulled from the following pages:
+```yaml {applies_to}
+serverless: unavailable
+```
```

Comment:
the way this label looks, it feels like it's just for serverless (a result of the coloring). imo I'd do a "stack" label instead

````suggestion
```yaml {applies_to}
stack:
```
````